### PR TITLE
Move signing from pipe to own module.

### PR DESCRIPTION
**Pull request by lauravuo**

### CODE REVIEW

File: std/didexchange/v0/invitation_impl.go [hunk 1]
```diff
@@ -111,7 +109,7 @@ func (m *invitationImpl) RoutingKeys() []string {
 	return m.Invitation.Services()[0].RoutingKeysAsB58()
 }
 
-func (m *invitationImpl) Verify(c crypto.Crypto, keyManager kms.KeyManager) error {
+func (m *invitationImpl) Verify(DID core.DID) error {
 	return nil
```
Comment (lainio) [inline std/didexchange/v0/invitation_impl.go:113]: Is it safe to return just nil?

File: std/didexchange/v0/request_impl.go [hunk 2]
```diff
@@ -134,7 +132,7 @@ func (m *requestImpl) RoutingKeys() []string {
 	return common.Service(m.Connection.DIDDoc, 0).RoutingKeys
 }
 
-func (m *requestImpl) Verify(c crypto.Crypto, keyManager kms.KeyManager) error {
+func (m *requestImpl) Verify(DID core.DID) error {
 	return nil
```
Comment (lainio) [inline std/didexchange/v0/request_impl.go:136]: Is it safe to return just `nil`?

**Review by lainio — APPROVED** [review]
Looks good. A couple of questions, though.

I also started to wonder, should we just start to use `latest` `golangci-lint`? At least for a moment, even if it could bring new linters and then new false positives. Maybe this is just temporary and they can fix everything that went broken after Go generics...

File: std/didexchange/v0/invitation_impl.go (hunk 1 above)
Comment (lauravuo) [reply std/didexchange/v0/invitation_impl.go:113]: Yes, there is nothing to verify for this message type.

**Pull request merged**